packages/windows/Dockerfile

```diff
@@ -0,0 +1,35 @@
+
+# https://docs.microsoft.com/en-us/visualstudio/install/build-tools-container advises tht dotnet is needed for build tools
+FROM mcr.microsoft.com/dotnet/framework/sdk:4.8-windowsservercore-ltsc2019 AS ci-windows-conan
+
+SHELL ["powershell.exe", "-ExecutionPolicy", "Bypass", "-Command"]
+
+ARG ASWF_CONAN_VERSION
+ARG ASWF_CONAN_PYTHON_VERSION
+ENV ASWF_CONAN_VERSION=${ASWF_CONAN_VERSION} \
+    ASWF_CONAN_PYTHON_VERSION=${ASWF_CONAN_PYTHON_VERSION}
+
+ENV chocolateyUseWindowsCompression=false \
+    PYTHONIOENCODING=UTF-8
+
+
+RUN iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1')); \
+    $env:Path += '";C:\tools\python3;C:\tools\python3\Scripts"'; \
+    choco install --no-progress --yes python3 --version=${Env:ASWF_CONAN_PYTHON_VERSION} --params '"/InstallDir:C:\tools\python3"'
+
+RUN choco install --no-progress --yes visualstudio2017buildtools --version=15.9.41.0
+RUN choco install --no-progress --yes visualstudio2017-workload-vctools --version=1.3.1
+RUN choco install --no-progress --yes --execution-timeout=0 visualstudio2017-workload-manageddesktop --version=1.2.1
```

Comment:
VFX Platform now specifies Visual Studio version: 2017 for CY2021, "Visual Studio 2019 v16.9 or later" for CY2022,
so version of Visual Studio to be pulled from Chocolatey should be in a more generic CY20XX config YAML?

Reply:
Good point, I've been trying to use the `ci-common` for the low-level toolchain, but the versions change at different rates for windows and linux, such a pain :(